recipe/meta.yaml

```
@@ -23,9 +23,11 @@ requirements:
     - pip
   run:
     - python >=3.7
-    - anndata
+    - anndata       # [py>37]
+    - anndata <0.9  # [py<=37]
     - attrs >=22.1
-    - numba
+    - numba >=0.57 # [py>37]
+    - numba <0.5.7 # [py<=37]
```

Should both of these be 0.57 or 0.5.7?

---

I think it should be `numba <0.57`

https://github.com/single-cell-data/SOMA/blob/063aef6a32530c2f2730c73942490d3fc55916d3/python-spec/requirements-py3.7.txt#L7

```suggestion
    - numba >=0.57 # [py>37]
    - numba <0.57 # [py<=37]
```